### ISSUE TITLE
squash consecutive spaces

### ISSUE THREAD

**Issue by missinglink** [pelias/whosonfirst#451]
Some place names contain multiple consecutive spaces, it should be safe to squash these down to a single space:

```
"county": "Division No.  6",
"county_gid": "whosonfirst:county:1158868963",
```

**Comment by orangejulius** [comment]
Definitely, this sort of thing is super annoying and causes confusion in the acceptance tests. Do you think we could take it one step further and _always_ perform this? Maybe in pelias/model? I can't think of a case where we would ever want consecutive spaces (or maybe even consecutive whitespace of any kind) in any of our text fields.
